[MI-2467] Updated documentation for pipeline features (#10)
Brightscout/mattermost-plugin-azure-devops: README.md

    /azuredevops repos subscription list anyone all_channels 
    ```

    - For listing Pipelines subscriptions

    ```
    /azuredevops pipelines subscription list anyone all_channels 
    ```

    Supported filters on the above slash command:
    - CreatedBy: `me`(show all subscriptions created by the current Mattermost user), `anyone`(show all subscriptions created by any Mattermost user)
    - Show for all channels: When the filter `all_channels` is passed in the slash command then subscriptions for all channels are listed. You can skip this filter param to list the subscriptions of the current channel only.

    **Note:** Only Mattermost users who are project admins or team admins on the linked Azure DevOps project can view/list subscriptions that exist in a channel where they are not a member.

- Approve pipeline requests from your channel: 
  - A user can approve pipeline requests from within the channel by subscribing to "Release deployment approval pending" or "Run stage waiting for approval" (for YAML pipelines) event notifications.

    ![image](https://user-images.githubusercontent.com/72438220/202639513-8fa58db0-bce9-46ed-864b-ebbae468a4b6.png)

>   - While the running stage or a release deployment is awaiting approval, a notification card is posted in the channel with the option to approve or reject the request. By reviewing the request's specifics in the notification, approvers can take the necessary action.

Reviewer: ```suggestion
  - While the running stage or a release deployment is awaiting approval, a notification is posted in the subscribed channel with the options to approve or reject the request. By reviewing the request's specifics in the notification, approvers can take the necessary action.
```

>   - When the running of a stage or a release deployment is awaiting approval, a notification card is posted in the channel with the option to approve or reject the request. Reviewing the request's specifics in the notification, approvers can take the necessary action.

Reviewer 2: ```suggestion
  - While the running stage or a release deployment is awaiting approval, a notification card is posted in the channel with the option to approve or reject the request. By reviewing the request's specifics in the notification, approvers can take the necessary action.
```

Reviewer 2: maybe?
